src/cache-save.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 The MathWorks, Inc.
+
+import * as core from '@actions/core';
+import * as cache from '@actions/cache';
+import {State} from './cache-state';
+
+export async function cacheMATLAB(useCache: string) {
+    if (useCache.toLowerCase() !== "true") {
+        return;
+    }
```

Comment:
This feels a little funny. It seems like `useCache` wants to be a boolean rather than a string. And this function shouldn't even be called if it is false.